packages/amplify-category-api/src/force-updates/force-refresh-schema.ts

```diff
@@ -0,0 +1,44 @@
+import path from 'path';
+import fs from 'fs-extra';
+import { getApiResourceDir } from './api-resource-paths';
+
+export const forceRefreshSchema = (): void => {
```

Comment:
This is just the functions from `auth-notifications.ts` which pertained to modifying the graphql schema, and exported as a new top-level method.